bot/resources/tags/local-file.md

```diff
@@ -0,0 +1,23 @@
+Thanks to discord.py, sending local files as embed images is simple. You have to create an instance of [`discord.File`](https://discordpy.readthedocs.io/en/latest/api.html#discord.File) class:
+```py
+# When you know the file exact path, you can pass it.
+file = discord.File("/this/is/path/to/my/file.png", filename="file.png")
+
+# When you have the file-like object, then you can pass this instead path.
+with open("/this/is/path/to/my/file.png", "rb") as f:
+    file = discord.File(f)
+```
+When using the file-like object, you have to open it in `rb` mode. Also, in this case, passing filename to it is not necessary.
```

Comment:
If "filename" isn't the name of a token of some kind, it should be "a file name". If it is a variable, it should be `filename` with backticks on either side. I see that it has backticks on the following line.

Reply:
I put this between backticks.